lib/api/gitlab.js

```diff
@@ -275,9 +284,14 @@ async function mergePr(pr) {
 // Generic File operations
 
 async function getFile(filePath, branchName = config.defaultBranch) {
-  const res = await glGot(
-    `projects/${config.repoName}/repository/files?file_path=${filePath}&ref=${branchName}`
-  );
+  // Gitlab API v3 support
+  let url;
+  if (config.apiVersion === 'v3') {
+    url = `projects/${config.repoName}/repository/files?file_path=${filePath}&ref=${branchName}`;
+  } else {
+    url = `projects/${config.repoName}/repository/files/${filePath}?ref=${branchName}`;
```

Comment:
According to the API docs, _filePath_ is expected to be a URL encoded string.

@rarkins based on your understanding of the `renovate` architecture, where would, or should, that URL encoding happen?